src/mixins/common-mixin.ts

```diff
@@ -68,4 +74,27 @@ export default class CommonMixin extends Vue {
     }
     return true
   }
+
+  /**
+   * Navigates to the dashboard page, optionally with a filing ID.
+   * @param identifier The identifier to include in the dashboard URL
+   * @param filingId The filing ID to include in the dashboard URL (optional)
+   */
+  protected navigateToDashboard (identifier: string, filingId?: number): void {
```

Comment:
You should assign a default filingId (eg, null or 0) so you can stay away from undefined, below.

Reply:
Updated